bazel.rc

```diff
@@ -0,0 +1,2 @@
+# Always include the version information in the build
+build --workspace_status_command=./bin/build_stamp.sh --action_env=ISTIO_VERSION --output_filter=^gogo
```

Comment:
Curious does this mean all warnings from gogo rules will be suppressed.. Is there any mechanism to suppress warnings just in a particular dir or the specific location where the target is called.

Reply:
It does mean that all warnings from gogo* rules will be suppressed. I believe that the filter is just for rule names (and not for directories). Hopefully, this will be just temporary.

I've sent a PR to fix the root cause (pubref/rules_protobuf#118).